internal/reports/REPORT.md

```diff
@@ -1,26 +1,26 @@
-## Harbourbridge Report
-A Harbourbridge report consists of elements defined below. A sample structured report can be found [here](/test_data/mysql_structured_report.json).
+## Spanner migration Report
+A Spanner migration report consists of elements defined below. A sample structured report can be found [here](/test_data/mysql_structured_report.json).
```

Comment:
Maybe "Spanner migration tool" report instead of "Spanner migration" report?

Reply:
Done.